### DISCUSSION TITLE
Update 3.9 --> 3.23

### DISCUSSION THREAD

**Discussion by ascheuch** [microsoft-search/pnp-modern-search#1648] [Q&A] [answered]
Hello,
it's possible to update Modern Search V3 directly from V 3.9 to V3.23?

Best Regards
Arnold

**Comment by wobba** [comment, marked as answer]
I suggest you try in a demo tenant, and see if it works. It's very likely there are breaking changes between those versions. If you read the release notes for all versions after 3.9 you will figure that out.

**Reply by ascheuch** [reply]
Thanks.
And thank you too for your work on this cool tool. 
It's my "Swiss Army knife' in SPOL.
